test/ctia/test_helpers/access_control.clj

```diff
@@ -951,86 +951,110 @@
                                                   (:parsed-body player-3-entity-repost2)]})))))))
 
 (defn test-access-control-tlp-settings
-  [entity new-entity]
-  (testing "TLP Settings Enforcement"
-    (swap! (p/global-properties-atom) assoc-in
-           [:ctia :access-control]
-           {:default-tlp "amber"
-            :min-tlp "amber"})
-
-    (let [{status-default-tlp :status
-           body-default-tlp :parsed-body}
-          (post (format "ctia/%s" entity)
-                :body (dissoc new-entity :tlp)
-                :headers {"Authorization" "player-1-token"})
-          {status-disallowed-tlp :status
-           body-disallowed-tlp :parsed-body}
-          (post (format "ctia/%s" entity)
-                :body (assoc new-entity :tlp "white")
-                :headers {"Authorization" "player-1-token"})]
-
-      (is (= 201 status-default-tlp))
-      (is (= "amber" (:tlp body-default-tlp)))
-
-
-      (is (= 400 status-disallowed-tlp))
-      (is (= "Invalid document TLP white, allowed TLPs are: amber,red"
-             (:message body-disallowed-tlp))))))
-
+  [entity new-entity fixtures]
+  (helpers/with-config-transformer*
+    #(assoc-in % [:ctia :access-control] {:default-tlp "amber"
+                                          :min-tlp "amber"})
+    #(fixtures
```

Comment:
wdyt of https://stuartsierra.com/2019/09/11/clojure-donts-short-fn-syntax ?

I'm aware we haven't discussed it, but for large cases like a 30-line `fixtures`, I'd say a threshold has definitely been surpassed. A large `#()` makes me visually hunt for `%` which may or not be inside the function body - awkward for a function so large.

Reply:
> A large #() makes me visually hunt for % which may or not be inside the function body

Good point. You also can't use a nested `#()` in those 30 lines.

> wdyt of https://stuartsierra.com/2019/09/11/clojure-donts-short-fn-syntax ?

Thanks for pointing me to it. I like the clarity of explaining the expansion of the `#()` reader macro. Personally don't really have much good to say about the rest of the post, as Stuart says in the first sentence, it seems written to those in their first 3 days of Clojure.

Reply:
Good catch.